docs/source/user_guide/automl.ipynb

```diff
@@ -47,7 +47,7 @@
    "source": [
     "__Note:__ To provide data to EvalML, it is recommended that you create a `DataTable` object using [the Woodwork project](https://woodwork.alteryx.com/en/stable/).\n",
     "\n",
-    "EvalML also accepts ``pandas`` input, and will run type inference on top of the input ``pandas`` data. If you’d like to change the types inferred by EvalML, you can use the `infer_feature_types` utility method as follows. The `infer_feature_types` utility method takes pandas or numpy input and converts it to a Woodwork data structure. It takes in a `feature_types` parameter which can be used to specify what types specific columns should be. In the example below, we specify that the provider, which would have otherwise been inferred as a column with natural language, is a categorical column."
+    "EvalML also accepts ``pandas`` input, and will run type inference on top of the input ``pandas`` data. If you\u2019d like to change the types inferred by EvalML, you can use the `infer_feature_types` utility method as follows. The `infer_feature_types` utility method takes pandas or numpy input and converts it to a Woodwork data structure. It takes in a `feature_types` parameter which can be used to specify what types specific columns should be. In the example below, we specify that the provider, which would have otherwise been inferred as a column with natural language, is a categorical column."
```

Comment:
Weird replacement here done by git but it looks fine: https://feature-labs-inc-evalml--1958.com.readthedocs.build/en/1958/user_guide/automl.html#AutoML-in-EvalML

Reply:
I've been seeing it come up in other PRs as well. I don't know what's up but agree it doesn't look like a problem.

Reply:
Same, looks fine though.

Reply:
That's so weird. This is [the unicode character "right single quotation mark"](https://www.fileformat.info/info/unicode/char/2019/index.htm) 😆  perhaps jupyter changed the way they represent unicode somehow.